Add: Menambahkan Cheat Sheet HTML5 & CSS, lainnya

putusujane opened the pull request:
Add: Menambahkan Cheat Sheet HTML5 & CSS, lainnya.

putusujane committed:
Add: Menambahkan Cheat Sheet HTML5 & CSS, lainnya

Menambahkan Cheat Sheet untuk HTML5 dan CSS.
Cheat Sheet merupakan shortcut atau jalan pintas penulisan kode atau tag HTML5 dan CSS di IDE.
IDE yang bisa digunakan Visual Studio Code, WebStorm, dan sejenisnya.

Menambahkan cara baik atau buruk penulisan tag atau kode HTML.
Disini dituliskan berbagai cara yang baik dan buruk penulisan kode atau tag HTML.

putusujane committed:
Merge pull request #2 from putusujane/putusujane-patch-1

Add: Menambahkan Cheat Sheet HTML5 & CSS, lainnya

putusujane committed:
Merge branch 'Alexandria-Library:main' into main

bhunivelze1 reviewed (changes requested):
Tiap link cheat sheetnya mungkin bisa ditambahkan emoji bendera di akhir kalimatnya biar jelas cheat sheetnya berbahasa apa. Biar konsisten dengan informasi-informasi yang di atas.

Emoji :indonesia: untuk bahasa Indonesia. Emoji :us: untuk bahasa Inggris.

putusujane committed:
Add: Menambah emoji bendera, menambah versi bahasa

Add:
- Menambah emoji bendera sesuai bahasa.
- Menambah versi bahasa Indonesia pada Cara Baik dan Buruk  Penulisan Tag/Kode HTML.

putusujane committed:
Edit: Memperbaiki link versi bahasa Indonesia

bhunivelze1 reviewed (approved):
Mantap :)
